### PR TITLE
Minor follow-ups to earlier pull requests

### PR DESCRIPTION
**Pull request by kgaillot**
This includes cherry-picking two commits from #1684

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this when you get a chance? Low priority

File: devel/README [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This directory contains helpers for the project developers.
+Everyone else can safely ignore it.
```
Comment (clumens) [inline devel/README:2]: This file should be added to EXTRA_DIST in devel/Makefile.am

Comment (kgaillot) [reply devel/README:2]: Thanks, updated

Comment (kgaillot) [reply devel/README:2]: Should we distribute gdbhelpers as well?

Comment (kgaillot) [reply devel/README:2]: Added gdbhelpers to EXTRA_DIST because why not, and tweaked devel/Makefile.am a bit

**Comment by clumens** [comment]
This all looks good to me.

**Pull request merged**